### PR TITLE
Added HomeFeedActivity.

### PR DESCRIPTION
**Pull request by jaindiv26**
**Description** I've modified the MainActivity. So now in this activity user have to enter the preferred Handel in order to see the contest list. Upon submitting the handel new activity is opened displaying the contest list.

**Issue link** #3 

**ScreenRecording**

https://drive.google.com/open?id=1E_NYoR81H0Y0BsLbe7wXlx5xEs5RR5QW

**Screenshots**

Screenshot_2019-10-03-23-47-15-551_com example android codeforces 

Screenshot_2019-10-03-23-47-05-860_com example android codeforces

Screenshot_2019-10-03-23-47-03-139_com example android codeforces

### CODE REVIEW

**Comment by jaindiv26** [comment]
@Chromicle There was no progress bar to show whether the list is loading or not. So I'm adding that currently and after that, this branch is ready to be merged. Please review the code and give a suggestion.

**Comment by jaindiv26** [comment]
@Chromicle It's done, I've added the determinate progress bar. Please check screen recording and screenshots.

**Comment by Chromicle** [comment]
Can you just solve merge conflicts

**Comment by jaindiv26** [comment]
Yeah, sure. Give me 5 minutes.

**Comment by jaindiv26** [comment]
@Chromicle Synced with the master branch.

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (Chromicle) [inline app/src/main/res/values/styles.xml]: why did you change this to no action bar 
for the main activity it is good if there is no action bar but please add the toolbar to the home feed activity

Comment (jaindiv26) [reply app/src/main/res/values/styles.xml]: Okay.

**Comment by jaindiv26** [comment]
@Chromicle I've added the Light action bar theme to HomeFeedActivity and displayed a toast when no contest was found.

Here's a screenshot of the changes I've made.

Screenshot_1570342281
